File: CS50/pset6/mario/more/mario.py
from cs50 import get_int

# user prompt
checker = True
while checker == True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        checker = False

# print pyramid
for i in range(0, height, 1):
    # print the space before the #
    for j in range(0, height - i - 1, 1):
        print(" ", end='')

    # print #
    for k in range(0, i + 1, 1):
        print("#", end='')

    # print "  "
    print("  ", end='')
    # print right side of pyramid
    for l in range(0, i + 1, 1):
        print("#", end='')

    # print /n
    print()

File: CS50/pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int h;
    // get h
    do
    {
        h = get_int("height: ");
    }
    while (h < 1 || h > 8);

    for (int i = 0; i < h; i++)
    {
        // print left side
        for (int k = 0; k < h - i - 1; k++)
        {
            printf(" ");
        }

        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        // space
        printf("  ");

        // print right side
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}